src/pages/Login/Login.tsx

```diff
@@ -33,7 +34,7 @@ const LoginPage: React.FC = () => {
   return (
     <AuthLayout>
       {isLoggedIn && <Redirect to="/boards" />}
-      {error && (
+      {error && showErrorMessage && (
```

Comment:
you set the showErrorMessage to true initially, and when the user edits the form, we set it to false. we then never set it to true again so no further error will be shown. Maybe you meant to set it to true in the submit handler?